src/lib.rs

```diff
@@ -258,3 +258,103 @@ impl Level {
         }
     }
 }
+
+#[cfg(feature = "zstd")]
+/// This module contains zstd-specific types for async-compression.
+pub mod zstd {
+    /// A compression parameter for zstd. This is a stable wrapper around zstd's own `CParameter`
+    /// type, to abstract over different versions of the zstd library.
+    ///
+    /// See the [zstd documentation](https://facebook.github.io/zstd/zstd_manual.html) for more
+    /// information on these parameters.
+    #[derive(Copy, Clone, Debug, PartialEq, Eq)]
+    #[non_exhaustive]
+    pub enum CParameter {
+        /// Compression level
```

Comment:
Aha sorry, I didn't know that they are behind `cfg(feature = "experimental")`.
I suppose I will submit a PR to zstd for that.

> I assumed you wouldn't want to do that. Happy to if you would like; I do like the rsyncable feature.

@robjtede Would it make sense to add an `experimental` feature in `async-compression` so that users can use these features?

Since we take on the task of creating a newtype wrapper of `zstd::stream::raw::CParameter`, I suppose we should also try to expose all underlying features if possible?

Reply:
@NobodyXu The Rust wrapper is reflecting the underlying instability in the zstd library; I think you'd need to convince the zstd C upstream to stabilize individual features.